src/test/run-make-fulldeps/extern-prelude/Makefile

```diff
@@ -0,0 +1,12 @@
+-include ../tools.mk
+
+all:
+	$(RUSTC) ep-lib.rs
+	$(RUSTC) ep-vec.rs
+
+	$(RUSTC) basic.rs --extern ep_lib=$(TMPDIR)/libep_lib.rlib
+	$(RUSTC) shadow-mod.rs --extern ep_lib=$(TMPDIR)/libep_lib.rlib
+	$(RUSTC) shadow-prelude.rs --extern Vec=$(TMPDIR)/libep_vec.rlib
+	$(RUSTC) feature-gate.rs --extern ep_lib=$(TMPDIR)/libep_lib.rlib 2>&1 | $(CGREP) "access to extern crates through prelude is experimental"
+	$(RUSTC) relative-only.rs --extern ep_lib=$(TMPDIR)/libep_lib.rlib 2>&1 | $(CGREP) "unresolved import"
+	$(RUSTC) relative-only.rs --extern ep_lib=$(TMPDIR)/libep_lib.rlib 2>&1 | $(CGREP) "failed to resolve"
```

Comment:
why is this using `run-make` and not auxiliary? ah, well, I guess we'd have to modify the `auxiliary` mechanism to supply `--extern`? (doesn't seem that hard... but maybe we leave it for another PR)